raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -849,6 +929,7 @@ private boolean handleBeginQuorumEpochResponse(
 
         Optional<Boolean> handled = maybeHandleCommonResponse(
             partitionError, responseLeaderId, responseEpoch, currentTimeMs);
+        // KAFKA-16529 will need to handle INVALID_VOTER_KEY
```

Comment:
I thought we already add in this PR? Or KAFKA-16529 should go first?

Reply:
This PR handles RPC requests and sends the appropriate responses. KAFKA-16529 will send the new version of the requests and handle the new responses. Let me update the comment to make this clear.